_includes/nightlies.md

```diff
@@ -0,0 +1,7 @@
+# [Nightlies](#Nightlies)
+
+Help development and test our nightlies. Add our F-Droid repository https://fdroid.storchp.de/fdroid/repo?fingerprint=BD08F441BE3C2D84B184230FE7EE4E1EEDEC321E75557548E56E1605C4DE9368 to your F-Droid client.
```

Comment:
When the link is a <a>, then one can also add the repo when visiting the page by phone

Reply:
Ok, I'll change it to an a href link.
After reading the docs again and Izzy's tutorial, I think I should create another repo key. I'll fix that first.

Reply:
The Link is changed.

Still thinking about the repo signing key. The only downside of the current key is the distinguished name which is set to "CN=manjaro, OU=F-Droid", by the fdroid CLI. I don't know if users would check and complain about it. I've never checked distinguished names of keys myself yet.

If I change it, users who already have the repo in their client would probably need to delete and readd it.

Reply:
Ok, decided to change the signing key before making the repo more public. Now or never. Now it has a meaningful distinguished name.